sort/tsh_sort_1_0.py

```diff
@@ -0,0 +1,64 @@
+#Some kind of a sort
+#The sort checks accounts for missing numbers
+#and last number in array being smaller
+#
+# Array to be sorted
+a = [10,2,6,7,3,5,8,1,13,15,4]
+#
+#Initalize variables
+minIndex = 0

```

Comment:
Alot of this could be simplified with for loops. 
https://www.tutorialspoint.com/python/python_for_loop.htm

The will allow you to initialize variables in the block of the loop, which will make it easier to follow / read:

```
for idx in range(len(a)):
   ....
   buffer = a[idx]
   a[idx] = newValue
   a[oldIdx] = buffer
```